core/src/main/scala/pickling/Macros.scala

```diff
@@ -31,6 +31,18 @@ trait TypeAnalysis extends Macro {
   }
 }
 
+// purpose of this macro: a wrapper around both pickler and unpickler
+trait PicklerUnpicklerMacros extends Macro {
+  import c.universe._
+
+  def impl[T: c.WeakTypeTag]: c.Tree = preferringAlternativeImplicits {
+    val tpe = weakTypeOf[T]
+    q"""
+      SPicklerUnpickler[$tpe](SPickler.generate[$tpe], Unpickler.generate[$tpe])
```

Comment:
Yep, including the `_root_` package.

q"""
_root_.scala.picking.SPicklerUnpickler[$tpe](_root_.scala.picking.SPickler.generate[$tpe], _root_.scala.picking.Unpickler.generate[$tpe])
"""

Another (although less commonly used) approach is to splice in a `reflect.universe.Symbol` in place of the name:

val SPicklerUnpickler: Symbol = symbolOf[SPicklerUnpickler] // Here the identifier is resolved according to this scope, not the macro expansion site

q"""$SPicklerUnpickler[$tpe](....)"""